.travis/install_from_source

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -e
+mkdir ./.binary/
+cd ./.binary/
```

Comment:
subshells can potentially help when doing `cd` comamnds - notably so you do not have to worry about returning back to the original directory.
EG:
```
+mkdir ./.binary/
(
 cd ./.binary/
../.travis/download_html_validator
 git clone https://github.com/w3c/css-validator.git
  (
    cd ./css-validator/
   git checkout 10409468e78e5aef25e11603c48dd8ac3b99a67d
   ant jar -q
  )
 (
  git clone https://github.com/RoiEXLab/link-checker.git
 cd ./link-checker/
 ./gradlew jar -q
  )
)
```
Not 100% sure if it makes it all that much nicer, but FWIW ^